src/Pages/Main/Cards.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import styled from 'styled-components';
+
+function Cards(props) {
```

Comment:
```suggestion
function Cards({bgImg, title, price, liketext ~~~~}) {
```
매개변수에서 바로 구조분해 가능합니다!

Reply:
수정했습니다!